leafsmart-back-end/index.js

```diff
@@ -15,13 +15,15 @@ const db = knex({
 });
 
 const favRoute = require("./routes/favouriteRouter");
+const savFavRoute = require('./routes/saveFavouritesRouter')
```

Comment:
I'm a little confused with these routes! Can we talk about them as a group? I'm confused about why all of the resource routes (create, read, delete) wouldn't all be in a single router file.

Reply:
We can clean up the routes in the back end, right now it's sorted around how the original requests were handled when testing. Understanding they are not super clean this was just getting everything working from backend request.